etna/articles/models.py

```diff
@@ -335,6 +337,8 @@ class RecordArticlePage(TopicalPageMixin, ContentWarningMixin, BasePageWithIntro
     # DataLayerMixin overrides
     gtm_content_group = "Record articles"
 
+    title_label = "RECORD REVEALED"
```

Comment:
Same as above - Would just attribute name `label` be sufficient - since its representing a name for that specific page?